types/smart-truncate/smart-truncate-tests.ts

```diff
@@ -1,4 +1,4 @@
-import { smartTruncate } from 'smart-truncate';
+import smartTruncate from 'smart-truncate';
```

Comment:
Use the commonjs syntax `import smartTruncate = require('smart-truncate')`. Also please remove esModuleInterop from tsconfig.

Consumers of the package who want to use `import smartTruncate from 'smart-truncate'` will need to add `"esModuleInterop": true` as you did, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.

Reply:
if I change first line to this:

```ts
const smartTruncate = require('smart-truncate');
```

I will get this error in tests:

```
Error: /workspace/DefinitelyTyped/types/smart-truncate/smart-truncate-tests.ts:1:23
ERROR: 1:23  no-var-requires  require statement not part of an import statement
ERROR: 1:23  expect           TypeScript@4.4 compile error: 
Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node` and then add 'node' to the types field in your tsconfig.
```

Are you sure I have to write `import smartTruncate = require('smart-truncate')`?

Reply:
OK I see this syntax for the first time but found the same at [`types/slice-source/slice-source-tests.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/slice-source/slice-source-tests.ts)